src/journey-maps/angular/journey-maps.ts

```diff
@@ -169,8 +169,11 @@ export class SbbJourneyMaps implements OnInit, AfterViewInit, OnDestroy, OnChang
   /** @docs-private */
   touchOverlayStyleClass: string = '';
 
+  /** In landscape mode, when the map height is below {@link SbbJourneyMaps#levelSwitchHorizontalThreshold},
```

Comment:
Nitpick, but please change the formatting to

```
/**
 * Text here
 */
```

Reply:
Done ✅